### ISSUE TITLE
Room Code Removal

### ISSUE THREAD

**Issue by profhupe** [mozilla/hubs#5187]
I noticed that the room code sharing option has been removed from Hubs. As someone who teaches with Hubs the room code sharing option is a lifesaver. My VR headsets have Hubs Link bookmarked so students only have to enter the code I provide during class. This has streamlined the process of getting students into VR environments. 

I understand, in theory, the reasons why the room codes would pose a problem, however, someone wanting to attack a room would have to test out a countless number of room codes before finding one that works and is also active at that time. 

Perhaps it would be possible to use alphanumeric codes for rooms, thus increasing the complexity of "discovering" a private room. 

In short, please bring back Room Code sharing.

**Comment by rawnsley** [comment]
[Related discussion](https://github.com/mozilla/hubs/discussions/5193).

@fonefen I would be interested to know if https://hmd.link works for you.

**Comment by fonefen** [comment]
> [Related discussion](https://github.com/mozilla/hubs/discussions/5193).
> 
> @fonefen I would be interested to know if https://hmd.link works for you.

Thank you very much for your contribution. For local connections it seems to be a solution for us. The problem is that our project is hybrid, we offer rooms for local but also remote connection. 
But thank you very much for informing us about the existence of hmd.link.

**Comment by fonefen** [comment]
Greetings to all. Is there any way to activate the code function in my custom project while working out ideas? It's been a nightmare for us to solve this problem. We have hybrid classes in our project and without code it is a very costly task for our clients. Can anyone help us? Sorry it sounds a bit desperate, but we are. We have a service in dozens of schools (we never had problems with the code at security level) and right now teachers are discarding to give classes with VR because of the difficulty that exists without the code option. We have tried the HMD.link option but not all schools are solving the problem. I hope someone can help us to retain the project. Thank you.

**Comment by matthewbcool** [comment]
Outside of hmd.link to send links locally another potential workaround (albeit requiring an extra step) might be making use of the favorite rooms feature. If the user has the room favorited from another device, it can appear on the home screen.
